### DISCUSSION TITLE
`space-x-8` not work well in rtl pages

### DISCUSSION THREAD

**Discussion by motahharm** [tailwindlabs/tailwindcss#7404] [Help] [answered]
![prob](https://user-images.githubusercontent.com/78138597/153347058-ee6d4c4a-5530-4965-8e87-88e86ddf59aa.png)

I want the `خانه` to be the same distance between `درباره ما` and `سرویس ها`.
But the `خانه` is attached to `درباره ما`

tailwind play:
https://play.tailwindcss.com/N4ucDkg3RE

Who can help me??it is a bug or not??

**Comment by adamwathan** [comment, marked as answer]
Use `space-x-reverse`: https://tailwindcss.com/docs/space#reversing-children-order

**Comment by motahharm** [comment]
@adamwathan 
Thanks.
